deny.toml

```diff
@@ -96,8 +84,6 @@ ignore = [
 # More documentation for the licenses section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
 [licenses]
-# The lint level for crates which do not have a detectable license
-unlicensed = "deny"
```

Comment:
Is that a new default or why don't we want it anymore?

Reply:
Apparently yes:

![Screenshot 2024-11-26 at 12-14-11 config - cargo-deny](https://github.com/user-attachments/assets/6eecefb7-be88-4787-89c4-9ebc49fb2ee0)